packages/ember-htmlbars/tests/helpers/closure_component_test.js

```diff
@@ -1,776 +0,0 @@
-  setup() {
-    owner = buildOwner();
-
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(component);
-    runDestroy(owner);
-    owner = component = null;
-  }
-});
-
-QUnit.test('renders with component helper', function() {
-  let expectedText = 'Hodi';
-  owner.register(
-    'template:components/-looked-up',
-    compile(expectedText)
-  );
-
-  let template = compile('{{component (component "-looked-up")}}');
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), expectedText, '-looked-up component rendered');
-});
-
-QUnit.test('renders with component helper with invocation params, hash', function() {
-  let LookedUp = Component.extend();
-  LookedUp.reopenClass({
-    positionalParams: ['name']
-  });
-  owner.register(
-    'component:-looked-up',
-    LookedUp
-  );
-  owner.register(
-    'template:components/-looked-up',
-    compile(`{{greeting}} {{name}}`)
-  );
-
-  let template = compile(
-    `{{component (component "-looked-up") "Hodari" greeting="Hodi"}}`
-  );
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), 'Hodi Hodari', '-looked-up component rendered');
-});
-
-QUnit.test('renders with component helper with curried params, hash', function() {
-  let LookedUp = Component.extend();
-  LookedUp.reopenClass({
-    positionalParams: ['name']
-  });
-  owner.register(
-    'component:-looked-up',
-    LookedUp
-  );
-  owner.register(
-    'template:components/-looked-up',
-    compile(`{{greeting}} {{name}}`)
-  );
-
-  let template = compile(
-    `{{component (component "-looked-up" "Hodari" greeting="Hodi") greeting="Hola"}}`
-  );
-  component = Component.extend({
-    [OWNER]: owner,
-    template
-  }).create();
-
-  runAppend(component);
-  equal(component.$().text(), 'Hola Hodari', '-looked-up component rendered');
-});
-
-QUnit.test('updates when component path is bound', function() {
-  let Mandarin = Component.extend();
-  owner.register(
-    'component:-mandarin',
-    Mandarin
-  );
-  owner.register(
-    'template:components/-mandarin',
-    compile(`ni hao`)
-  );
-  owner.register(
-    'template:components/-hindi',
-    compile(`Namaste`)
-  );
-
-  let template = compile('{{component (component lookupComponent)}}');
-  component = Component.extend({
-    [OWNER]: owner,
-    template,
-    lookupComponent: '-mandarin'
-  }).create();
-
-  runAppend(component);
-
-  equal(component.$().text(), `ni hao`,
-        'mandarin lookupComponent renders greeting');
-  run(() => {
-    component.set('lookupComponent', '-hindi');
-  });
-  equal(component.$().text(), `Namaste`,
-        'hindi lookupComponent renders greeting');
-});
-
-QUnit.test('updates when curried hash argument is bound', function() {
```

Comment:
Will be added in the next pushed. It might take a couple of hours to modify every test.